### ISSUE TITLE
Update mica to 4.21.0 

### ISSUE THREAD

**Issue by jeanconn** [sot/skare3#319]
Update mica to 4.21.0 in ska3-flight.

https://github.com/sot/mica/releases/tag/4.21.0

This release supports the changes introduced in DS 10.8.3 and is somewhat urgent (our trending is not working until we promote and use mica 4.21.0 in HEAD ska3/flight ).

**Comment by javierggt** [comment]
@jeanconn, what is the release/PR info from mica that corresponds to this?

**Comment by jeanconn** [comment]
Not sure about the question, but the https://github.com/sot/mica/releases/tag/4.21.0 should finally get us going after DS 10.8.3 promotion.

**Comment by javierggt** [comment]
I'm afraid this does not contain enough information. From the perspective of skare3, the questions are:
- What mica release should one add to skare3
- What is the release information attached to it
- Is there something that should happen before this ends up in ska3-flight or other?

I do not know enough to tell what "after DS 10.8.3 promotion" means and how it is relevant for the skare3 release process.

**Comment by javierggt** [comment]
In other words: so... mica 4.21.0 is the one that should go into skare3 2020.3 release? are we clear to proceed?

**Comment by jeanconn** [comment]
Yes.  

The action was opened here when CXCDS promoted a software release external to our software (DS 10.8.3) that changed the format of ASP L1 data in the archive.  At that point we needed changes in mica to deal with those new files (interface change) so I opened an issue to make sure not to forget about it. Whether that issue should have been a mica issue, a ska_projects issue, or a skare3 issue is somewhat moot.  Those changes are now in mica 4.21.0 so including that release closes this "system" issue.  I think the release information is in that release per your auto release annotation.

Mica 4.21.0 also includes changes to the columns in guide stats.  The current processing job to update that file is not working anyway, so I've just moved the new "with mag stats" file to be the "official" guide_stats.h5 file and it will be updated when the mica 4.21.0 code is promoted and cron job started.  So that is the "is there something that should happen before this ends up in ska3-flight" action and is complete.
